CommonClient.py

```diff
@@ -182,6 +182,7 @@ class CommonContext:
     starting_reconnect_delay: int = 5
     current_reconnect_delay: int = starting_reconnect_delay
     command_processor: typing.Type[CommandProcessor] = ClientCommandProcessor
+    ui_manager: typing.Optional["kvui.GameManager"] = None
```

Comment:
```suggestion
    ui_manager: typing.Optional["type[kvui.GameManager]"] = None
```
`ui_manager` is not an instance of `GameManager` - It's a subclass of `GameManager`